src/components/CitySelectionForm.js

```diff
@@ -0,0 +1,45 @@
+import "./../styles/CitySelectionForm.css";
+import { useState } from "react";
+
+function CitySelectionForm(props) {
+  const [userChoice, setUserChoice] = useState("placeholder");
+  // console.log("Inicial Users Choice State:", userChoice); //RETURNS placeholder  - OK
+
+  //Get the User's choice value
+  const handleUserChoice = (e) => {
+    console.log("City from Form:", e.target.value);
+    //Update userChoice
+    setUserChoice(e.target.value);
+    // console.log("Users Choice State:", userChoice); //Why 1 behind???
```

Comment:
Not sure if this is still being used, but if not then the commented code can be removed